src/main/java/org/fundacion/jala/converter/view/Models/ExtractTextRequestForm.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Daniela Santa Cruz Andrade
+ * @version 1.0
```

Comment:
We agreed not to put the version. It should be deleted

Reply:
Ok, already fixed